app-admin/passwordsafe/passwordsafe-1.11.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit cmake desktop flag-o-matic wxwidgets
+
+MY_PV="${PV/_beta/BETA}"
+DESCRIPTION="Password manager with wxGTK based frontend"
+HOMEPAGE="https://pwsafe.org/ https://github.com/pwsafe/pwsafe/"
+SRC_URI="https://github.com/pwsafe/pwsafe/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Artistic-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="libressl minimal qr test +xml xvkbd yubikey"
```

Comment:
`xvkbd` doesn't seem to have function in this ebuild, so put it as `optfeature` instead.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

Also `minimal` seems rather redundant.

Reply:
Ok, will move `USE=xvkbd` to `optfeature` (never heard of that before....).
About `USE=minimal` - explanation ist in `src_prepare`:
```
    # binary name pwsafe is in use by app-misc/pwsafe, we use passwordsafe
    # instead. Perform required changes in linking files

```
an USE=minimal is installing only "/usr/bin/passwordsafe" where USE=-minimal installs /usr/bin/pwsafe and a symlink passwordsafe->pwsafe.

This as explanation - my opinion is that taking care about app-misc/pwsafe is obsolete ab done in the wrong way, because 

- the pwsafe source is unreachable and can be considered dead? (no release since years)
- current passwordsafe database format is probably not supported by app-misc/pwsafe
- IMHO pwsafe binary should be renamed to pwsafe-cli 

As a result, I will remove USE=minimal, leave DEPEND=!app-misc/pwsafe and install /usr/bin/passwordsafe - to be consistent with the package name.

Reply:
@juippis I am re-thinking the binary file rename behaviour... because I discovered there is a `pwsafe-cli` included. Seems that keeping binary names pwsafe and pwsafe-cli makes sense, and a symlink passwordsafe->pwsafe for backward compatibility.